baselines/pFedHN/pFedHN/utils.py

```diff
@@ -0,0 +1,70 @@
+"""Define any utility function.
+
+They are not directly relevant to  the other (more FL specific) python modules. For
+example, you may define here things like: loading a model from a checkpoint, saving
+results, plotting.
+"""
+import json
+import random
+
+import numpy as np
+import torch
+from matplotlib import pyplot as plt
+
+
+##########################################################
+# UTILS #
+##########################################################
+def set_seed(seed):
+    """Set the seed."""
+    random.seed(seed)
+    np.random.seed(seed)
+    torch.manual_seed(seed)
+
+
+def get_device(no_cuda=False, gpus="0"):
+    """Get the device."""
+    return torch.device(
+        f"cuda:{gpus}" if torch.cuda.is_available() and not no_cuda else "cpu"
+    )
```

Comment:
```suggestion
```

Reply:
remove (based on all my previous comments for server and client)

Reply:
Taken care